nodes/mainnet.md

```diff
@@ -76,6 +76,9 @@ consensus full node with no pruning for your bridge node.
 
 RPCs for DA nodes to initialise or start your celestia-node to Mainnet Beta with:
 
+- `public-celestia-da.numia.xyz`
```

Comment:
still getting an issue accessing state

```bash
celestia state balance --token $AUTH_TOKEN
{
  "result": {
    "code": 1,
    "message": "error in json rpc client, with http response metadata: (Status: 503 Service Unavailable, Protocol HTTP/1.1). error unmarshalling: invalid character '\u003c' looking for beginning of value"
  }
}
```

Reply:
`celestia light start --core.ip public-celestia-consensus.numia.xyz` - works as expected, with successful state query

`celestia light start --core.ip public-celestia-da.numia.xyz` - times out https://app.warp.dev/block/OOjwwOOwBFKnc4HBNgwvK0

Reply:
the consensus urls should work for light/bridge nodes.
the da url moved, too and leads now to the bridge node rpc and should work for requests like `celestia state balance...`